internal/brokers/testdata/TestNewManager/golden/successfully_create_manager_identifying_brokers

```diff
@@ -0,0 +1,4 @@
+- local
+- Broker
+- broker foo
```

Comment:
I think broker foo and bar are only temporary. I would have multiple dbus broker instead with their respective configuration too.

Reply:
You guys are still using the local brokers to run some local tests, so that's why I kept them. I can comment the code with a //TODO so that it's still there for manual tests and it does not show on the tests if that's ok.

Reply:
See my comment about global env variables